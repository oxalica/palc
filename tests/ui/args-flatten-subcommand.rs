use palc::{Args, Subcommand};

#[derive(Args)]
struct Outer {
    #[command(flatten)]
    inner: Inner,
}

#[derive(Args)]
struct Inner {
    #[command(subcommand)]
    subcmd: Subcmd,
}

#[derive(Subcommand)]
enum Subcmd {}

fn main() {}
