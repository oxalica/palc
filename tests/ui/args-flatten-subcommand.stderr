error[E0080]: evaluation panicked: cannot flatten an Args with subcommand
 --> tests/ui/args-flatten-subcommand.rs:6:12
  |
6 |     inner: Inner,
  |            ^^^^^ evaluation of `_` failed here
